packages/ui/components/dialog/Dialog.tsx

```diff
@@ -89,7 +89,7 @@ export const DialogContent = React.forwardRef<HTMLDivElement, DialogContentProps
         <DialogPrimitive.Content
           {...props}
           className={classNames(
-            "fadeIn bg-default scroll-bar fixed left-1/2 top-1/2 z-50 w-full max-w-[22rem] -translate-x-1/2 -translate-y-1/2 rounded-md text-left shadow-xl focus-visible:outline-none sm:align-middle",
+            "fadeIn bg-default scroll-bar fixed left-1/2 top-1/2 z-50 max-w-[22rem] -translate-x-1/2 -translate-y-1/2 rounded-md text-left shadow-xl focus-visible:outline-none sm:align-middle",
```

Comment:
this could have side effects and need to be careful with

Reply:
> this could have side effects and need to be careful with

I was into the same problem while making changes over here.
The issue was that the dialog-creation class was being created dynamically and the <form> was taking the width of the dialog-creation.
<form> had datepPicker and an area where selected dates will be shown. That's why I made a change that dialog-creation should not take full width always. It should take full width if there is content to display.
And this solved our issue. 
Initially, we had nothing in the right side of datePicker that's why the size of the dialog-creation gets reduced to only the width of datePicker.

Most importantly, it is not creating size issues over other places because it will take the full size if there is content to display. I hope it makes sense.
By the way I did checked with another popups available in the app, all were working fine as expected.